PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,39 @@
+# Pull Request Template
+
+## Description
+
+Describe the changes that where made in this pull request. When possible start with a user story - short, simple descriptions of a feature told from the perspective of the person who desires the new capability. Be sure to also include the following information:
+
+
+1. Motivation for change
+2. What was changed
+3. Link to relevant issues and documentation
+
+Example:
+*As a Blockstack developer, I would like to encrypt files using the app private key. This is needed because storing unencrypted files is unacceptable. This pull request adds the* `*encryptContent*` *function which will take a string and encrypt it using the app private key. For details refer to issue #123* 
+
+
+## Type of Change
+- [ ] New feature
+- [ ] Bug fix
+- [ ] Documentation update
```

Comment:
Documentation update -> API reference change

Documentation is about what's in docs.blockstack.org -- the content in this repo is source for the generated API reference only.

Reply:
I think this will be a good place to bring attention to required changes on doc.blockstack.org.